src/Test.UnitTests.Sarif/JsonTests.cs

```diff
@@ -18,8 +18,6 @@ public abstract class JsonTests
         protected static readonly Tool DefaultTool = new Tool() { Driver = new ToolComponent { Name = "DefaultTool" } };
         protected static readonly Result DefaultResult = new Result { Message = new Message { Text = "Some testing occurred." } };
 
-        protected static readonly ResourceExtractor ResourceExtractor = new ResourceExtractor(typeof(JsonTests));
```

Comment:
>ResourceExtractor [](http://example.com/codeflow?start=52&length=17)

I removed this in preference of cleaning up the single derived type that needs test extraction.